src/utils.ts

```diff
@@ -676,7 +676,13 @@ export function prevString(s: string, alphabet = DEFAULT_ALPHABET): string {
 export function lexicographicCompare(a: string, b: string): number {
     // Dev note: this exists because I'm sad that you can use math operators on strings, so I've
     // hidden the operation in this function.
-    return (a < b) ? -1 : ((a === b) ? 0 : 1);
+    if (a < b) {
+        return -1;
+    } else if (a > b) {
+        return 1;
+    } else {
+        return 0;
+    }
```

Comment:
I mean, sure. The code was clear enough before though I think.

Reply:
It got upset about nested ternaries, and I don't blame it